src/test/java/de/tum/in/www1/artemis/exam/ExerciseGroupIntegrationTest.java

```diff
@@ -163,25 +164,20 @@ void testDeleteExerciseGroup_asEditor() throws Exception {
     void importExerciseGroup_successfulWithExercisesIntoSameExam() throws Exception {
         Exam targetExam = examUtilService.addExamWithModellingAndTextAndFileUploadAndQuizAndEmptyGroup(course1);
 
-        final List<ExerciseGroup> listExpected = targetExam.getExerciseGroups();
+        final List<ExerciseGroup> exerciseGroupsBefore = targetExam.getExerciseGroups();
 
-        final List<ExerciseGroup> listReceived = request.postListWithResponseBody("/api/courses/" + course1.getId() + "/exams/" + targetExam.getId() + "/import-exercise-group",
-                listExpected, ExerciseGroup.class, HttpStatus.OK);
-        assertThat(listReceived).hasSize(9);
+        final List<ExerciseGroup> exerciseGroupsNow = request.postListWithResponseBody(
+                "/api/courses/" + course1.getId() + "/exams/" + targetExam.getId() + "/import-exercise-group", exerciseGroupsBefore, ExerciseGroup.class, HttpStatus.OK);
 
-        // We import into the same exam -> double the exercise groups
-        listExpected.addAll(listExpected);
+        assertThat(exerciseGroupsNow).hasSize(9);
+        assertThat(exerciseGroupsNow).containsAll(exerciseGroupsBefore);
+        assertThat(exerciseGroupsNow).allMatch(element -> element.getId() != null);
```

Comment:
You can combine assertions of the same collection into one call.
```suggestion
        assertThat(exerciseGroupsNow).hasSize(9).containsAll(exerciseGroupsBefore).allMatch(element -> element.getId() != null);
```